pkg/pillar/cmd/zedagent/reportinfo.go

```diff
@@ -669,3 +679,31 @@ func createAppInstances(ctxPtr *zedagentContext,
 	ctxPtr.getconfigCtx.subAppInstanceStatus.Iterate(
 		addAppInstanceFunc)
 }
+
+func getSMARTAttributeValue(field string) int64 {
+	filename := "/run/SMART_details"
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		log.Errorf("getSMARTAttributeValue: exception while opening %s. %s", filename, err.Error())
```

Comment:
Does this mean an error log every 30 seconds when we have an nvme disk? That's bad. Doing it once is fine.

Reply:
Doing it once as part of a goroutine.